README.md

```diff
@@ -223,12 +223,15 @@ from qiskit.aqua.components.optimizers import L_BFGS_B
 optimizer = L_BFGS_B()
 
 # setup the initial state for the variational form
-from qiskit.chemistry.components.initial_states import HartreeFock
+from qiskit.chemistry.circuit.library import HartreeFock
 init_state = HartreeFock(num_spin_orbitals, num_particles)
 
 # setup the variational form for VQE
 from qiskit.circuit.library import TwoLocal
-var_form = TwoLocal(num_qubits, ['ry', 'rz'], 'cz', initial_state=init_state)
+var_form = TwoLocal(num_qubits, ['ry', 'rz'], 'cz')
+
+# add the initial state
+var_form.compose(init_state, front=True)
```

Comment:
So this is a required change in that initial_state will not accommodate a QuantumCircuit? If it is now done this way it seems the var form will not have visibility to the initial state as it had when it was passed directly and it would prepend it itself.

Reply:
We had a workaround in place where the NLocal circuits expected a `InitialState`. I'll add support for passing a QuantumCircuit and deprecate the InitialStates once this is merged. We can update the readme sample to use the kwarg again.